src/storage/hnsw.cpp

```diff
@@ -20,26 +20,50 @@
 // hnsw.cpp - implementation of HNSW index
 //
 
-#include <bits/stdc++.h>
-
 #include "hnsw.hpp"
 
 // TODO : When cub_server terminates, hnsw_index_id will be reset to 0.
 //        This is not a problem in current implementation, but it may be a problem in the future,
 //        such as duplicate hnsw_index_id when cub_server restarts.
 //        We need to consider a better way to identify the hnsw index.
 int hnsw_index_id = 0;
+std::unordered_map<int, faiss::IndexHNSW *> hnsw_index_map;
```

Comment:
```suggestion
std::unordered_map<int, std::unique_ptr<faiss::IndexHNSW>> hnsw_index_map;
```
위와 같이 정의 후에 `delete it->second;` 없이 hnsw_index_map.erase () 로 해제할 수 있을 것 같습니다.

Reply:
효율적인 메모리 관리를 위해 스마트 포인터를 사용하는 것은 좋아보입니다. 반영하였습니다.